samples/hello-world-frontend-plugin.devfile

```diff
@@ -0,0 +1,29 @@
+specVersion: 0.0.1
+name: theia-hello-world-plugin
```

Comment:
Maybe it makes sense to rename sample file name to make it clear that it is Devfile for **Theia** frontend plugin?

Reply:
yes